### PR TITLE
guix: Make V=1 more powerful for debugging

### PR DESCRIPTION
**Pull request by dongcarl**
```
- Print commands in both unexpanded and expanded forms
- Set VERBOSE=1 for CMake
```

Ping MarcoFalke hopefully you use `V=1` already for the Guix builds on DrahtBot?

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Thanks! Concept A⃣ C⃣ K⃣

File: contrib/guix/libexec/build.sh [hunk 1]
```diff
@@ -3,6 +3,14 @@ export LC_ALL=C
 set -e -o pipefail
 export TZ=UTC
 
+if [ -n "$V" ]; then
```
Comment (laanwj) [inline contrib/guix/libexec/build.sh:6]: No idea if it's a problem, but won't this detect V=0 as well?

Comment (dongcarl) [reply contrib/guix/libexec/build.sh:6]: Right yes, I just wanted to conform with the `README` which says that we will do thing if "$V" is non-empty. Perhaps I can note in the `README` that we don't try to interpret the value of "$V" at all.

**Comment by MarcoFalke** [comment]
𝒜𝒞𝒦 82f4b5a98e2ab213f02773eed17019df23f731cd

File: contrib/guix/README.md [hunk 2]
```diff
@@ -141,6 +141,11 @@ find output/ -type f -print0 | sort -z | xargs -r0 sha256sum
 
   If non-empty, will pass `V=1` to all `make` invocations, making `make` output
   verbose.
+
```
Comment (MarcoFalke) [inline contrib/guix/README.md:144]:
```suggestion

```

**Review by fanquake — APPROVED** [review]
ACK f852761aec81ed23c7b9e4546c08d1ef303f2507. Ran a Windows Guix build and compared the output from master and this PR when using `V=1`. i.e `HOSTS=x86_64-w64-mingw32 PATH="/root/.config/guix/current/bin${PATH:+:}$PATH" V=1 ./contrib/guix/guix-build.sh`.

**Pull request merged**